tests/plotting/test_plot_manager.py

```diff
@@ -22,8 +22,12 @@
 
 log = logging.getLogger(__name__)
 
-expected_result: PlotRefreshResult = PlotRefreshResult()
-expected_result_matched = True
+
+class TestRefreshResult:
+    loaded: int = 0
+    removed: int = 0
+    processed: int = 0
+    remaining: int = 0
```

Comment:
it seems like using the `PlotRefreshResult` type in the test would be a good opportunity to test this new feature. Now it seems there's no test for it.

Reply:
Make sense, see 0f67b80.